src/odin/server/traffic.rs

```diff
@@ -0,0 +1,26 @@
+use std::process::Child;
+
+use pcap::{Capture, Device};
+
+pub fn spawn_kill_on_idle(child: &mut Child, pause_on_idle_s: u32) -> Result<(), String> {
```

Comment:
well, the plan was to _pause_ the process. i realized... i'd need _another_ cgroup to _pause_ anything, and i'm not _yet_ talented enough to do that. so, my first pass was going to be naively killing the pid, then rebooting it on traffic. then on a second pass, try the whole pause/resume bit, once i can learn about firing up that server in a pausable cgroup.

Reply:
arrrg i'm such a dummy! the proc does respond to SIGSTOP/SIGCONT already. no cgroups required